plugins/opds.koplugin/main.lua

```diff
@@ -23,7 +23,12 @@ function OPDS:showCatalog()
     local OPDSCatalog = require("opdscatalog")
     local filemanagerRefresh = function() self.ui:onRefresh() end
     function OPDSCatalog:onClose()
-        filemanagerRefresh()
+        local FileManager = require("apps/filemanager/filemanager")
+        if FileManager.instance then
+            filemanagerRefresh()
+        else
+            FileManager:showFiles(G_reader_settings:readSetting("download_dir"))
+        end
         UIManager:close(self)
```

Comment:
Could you move that *before* the FM shenanigans while you're there?

(It's unlikely to change anything in practice, it's just a more logical ordering).

Reply:
Sorry, couldn't get where to move?

Reply:
I.e., add the new lines below this one rather than above.